frontend/src/components/form/StepperNavigation.vue

```diff
@@ -39,20 +39,20 @@ function handlePrevClick() {
 <template>
   <div class="flex pt-4 justify-content-between">
     <Button
+      :style="{ visibility: props.prevDisabled ? 'hidden' : 'visible' }"
       class="px-4 py-1"
       outlined
       icon="pi pi-arrow-left"
       icon-class="text-xl"
-      :disabled="props.prevDisabled"
```

Comment:
Removing this makes button clickable while disabled, even if not visible through `visibility: hidden`.